### PR TITLE
chore: replace easygettext with scripts from vue3 gettext

### LINKED ISSUE #10249: Get rid of `easygettext` and use `vue3-gettext` instead
## Is your feature request related to a problem? Please describe.
We currently use `easygettext` for extracting our language strings so they can be imported into Transifex. That's not optimal because we needed to fork the lib and therefore can't easily bump it if needed (like currently because of some security alerts).

## Describe the solution you'd like
Since we're already using `vue3-gettext` we can and should use the extractor from that lib.

### PR DESCRIPTION
**Pull request by LukasHirt**
## Description

- use vue3 gettext to extract and compile translations
- introduce gettext config which configures extraction and compilation
- drop LC_MESSAGES from path of translation files (needed due to configuration of extraction and compilation where we cannot specify path after locale)

## Related Issue

- resolves #10249

## Motivation and Context

Get rid of dep alerts

## How Has This Been Tested?

- test environment: macos x64
- test case 1: makefile

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests
- [ ] Documentation
- [x] Maintenance (e.g. dependency updates or tooling)

### CODE REVIEW

**Comment by LukasHirt** [comment]
@JammingBen @dschmidt please let me know how would you like to approach changelog here

**Comment by dschmidt** [comment]
no change log pretty much irrelevant for users and external developers

File: packages/design-system/docs/i18n.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Writing translations
 
-For translations we are using combination of [vue-gettext](https://github.com/Polyconseil/vue-gettext) and [easygettext](https://github.com/polyconseil/easygettext).
+For translations we are using combination of [Vue 3 Gettext](https://jshmrtn.github.io/vue3-gettext/).
```
Comment (dschmidt) [inline packages/design-system/docs/i18n.md:3]: no combination

File: packages/web-app-external/src/App.vue [hunk 2]
```diff
@@ -105,7 +104,7 @@ export default defineComponent({
 
         const query = stringify({
           file_id: fileId,
-          lang: language.current,
+          lang: currentLanguage,
```
Comment (dschmidt) [inline packages/web-app-external/src/App.vue:107]: This was probably the reason why there was a language object and the return value of useGettext() wasn't destructured ... this loses reactivity

File: packages/web-runtime/src/container/sse/files.ts [hunk 3]
```diff
@@ -153,13 +153,14 @@ export const onSSEItemTrashedEvent = ({
     // If initiated by current client (browser tab), action unnecessary. Web manages its own logic, return early.
     return
   }
+  const { $gettext } = language
```
Comment (dschmidt) [inline packages/web-runtime/src/container/sse/files.ts:156]: if this works, you can use this approach above where you need to access `language.current`

**Review by dschmidt — CHANGES_REQUESTED** [review]

File: packages/web-app-external/src/App.vue (hunk 2 above)
Comment (kulmann) [reply packages/web-app-external/src/App.vue]: yep, sure. Could you check the PR again please?

**Pull request merged**